agent/command.go

```diff
@@ -215,6 +215,12 @@ func (a *Agent) runCommandOrFunc(ctx context.Context, tc *taskContext, commandIn
 			commandSpan.SetStatus(codes.Error, "running command")
 			commandSpan.RecordError(err, trace.WithAttributes(tc.taskConfig.TaskAttributes()...))
 			commandSpan.End()
+			if commandInfo.RetryOnFailure {
+				logger.Task().Info("Command is set to retry on failure, marking task to restart upon completion.")
```

Comment:
retry once** on failure perhaps? Just so its clear why we wouldn't see this another time

Reply:
Updated it to a formatted string that references `MaxAutomaticRestarts` as this feature's been set up to change the number of allowed restarts moving forward if necesary